cpp/include/Ice/Proxy.h

```diff
@@ -187,7 +201,7 @@ class InvokeLambdaOutgoing : public InvokeOutgoingAsyncT<R>, public LambdaInvoke
             {
                 if(this->_is.b.empty())
                 {
-                    response(R { ok, { 0, 0 }});
+                    response(makeIceInvokeResult<R>(true, 0, 0));
```

Comment:
Using `{}` seems to produce an empty pair {0, 0} and an empty vector size 0.

Reply:
So we only need makeIceInvokeResult<T> for createProxy?

It would be good to add a test - I am wondering if `{ 0, 0 }` is only problematic with Ice_invokeResult because of the 0 literals. When using variables with proper types, we should always get constructor (5) - and never constructor (3) or (10).

Reply:
> Using {} seems to produce an empty pair {0, 0} and an empty vector size 0.

It's actually not clear to me that `{ }` is correct to create a `{ nullptr, nullptr }` pair<Byte*>. What about using `{nullptr, nullptr}`. This is the clearest; the compiler won't confuse nullptr with a byte value.